torch/_tensor.py

```diff
@@ -366,6 +366,14 @@ def _reduce_ex_internal(self, proto):
                 ),
             )
             return (torch._utils._rebuild_sparse_tensor, args_sparse_compressed)
+        elif self.is_nested:
+            args_nested = (
+                self.values(),
```

Comment:
This won't work for arbitrary NestedTensors though right?

Reply:
Ok true, `values()` has a contiguity restriction :(

Reply:
Discussed offline: eventually introduce a private API that does what `values()` does now and deprecate `values()` in its current form, to be replaced by appropriate usage of `view(-1)` / `reshape(-1)`.